Holovibes/includes/API.hxx

```diff
@@ -620,34 +620,23 @@ inline void set_vesselness_sigma(double value)
 inline int get_min_mask_area() { return GET_SETTING(MinMaskArea); }
 inline void set_min_mask_area(int value) { return UPDATE_SETTING(MinMaskArea, value); }
 
-inline OtsuKind get_otsu_kind() { return GET_SETTING(OtsuKind); }
-inline void set_otsu_kind(OtsuKind value) { return UPDATE_SETTING(OtsuKind, value); }
-inline int get_otsu_window_size() { return GET_SETTING(OtsuWindowSize); }
-inline void set_otsu_window_size(int value) { UPDATE_SETTING(OtsuWindowSize, value); }
-inline float get_otsu_local_threshold() { return GET_SETTING(OtsuLocalThreshold); }
-inline void set_otsu_local_threshold(float value) { UPDATE_SETTING(OtsuLocalThreshold, value); }
-
-/*!
- * \name Bwareafilt
- *
- */
-inline bool get_bwareafilt_enabled() { return GET_SETTING(BwareafiltEnabled); }
-inline void set_bwareafilt_enabled(bool value)
+inline float get_diaphragm_factor() { return GET_SETTING(DiaphragmFactor); }
+inline void set_diaphragm_factor(float value)
 {
-    UPDATE_SETTING(BwareafiltEnabled, value);
-    pipe_refresh();
+    value = value > 1.0f ? 1.0f : (value < 0 ? 0 : value);
```

Comment:
Is the negative check necessary ? I don't recall it being done in any other setter

Reply:
Well my reasoning is 'you never know'
At least here the value is guaranteed to be correct idk
Maybe it will be useful once the back is a .dll and these functions are callable